lib/Site.js

```diff
@@ -17,6 +19,7 @@ const BOILERPLATE_FOLDER_NAME = '_boilerplates';
 const SITE_ASSET_FOLDER_NAME = 'asset';
 const TEMPLATE_ROOT_FOLDER_NAME = 'template';
 const TEMPLATE_SITE_ASSET_FOLDER_NAME = 'markbind';
+const USER_VARIABLES_PATH = '_system/variables.md';
```

Comment:
It's odd that `_system` contains user-defined variables 😕

Reply:
Any suggestion? Just use `_variables.md`?

Reply:
Better to be in a folder. How about `_markbind/variables.md`? (cc: @damithc)

Reply:
@damithc Any thoughts?

Reply:
both boilerplates are variables are user defined.
So, `_markbind/_boilerplates` and `_markbind/_variables.md`?
Or, `_markbind/boilerplates` and `_markbind/variables.md`?

Reply:
I prefer the second. @Gisonrg Any thoughts?

Reply:
Yea the second one makes sense to me, since we already use"_" for the folder name there is no need to add it again for files inside it.